Recursion/Print_Decrease_Increase_of_a_number.cpp

```diff
@@ -0,0 +1,51 @@
+// Write a program that will print a number first in decrease order then in increase order using recursion.
+#include <iostream>
+class PrintDecreaseIncrease{
+    private:
+        int n;
+        //Recursive function 
+        void pdi(int n){
```

Comment:
Pass the parameter by reference instead of by value in the recursive function. Do you know the number of ways parameters can be passed in C++? How is it different from c? 

Something like this.

```
void printDecreaseIncrease(int &num) {
  if (num == 0) 
	  return;
  std::cout << num << std::endl;
  printDecreaseIncrease(--num);
  std::cout << num + 1 << std::endl;
}
```